### PR TITLE
Update UI (slider and etc.)

### PR DESCRIPTION
**Pull request by dooho-lee**
1. fix slider ui more stick and more colorful
2. add dimming spinbox and colortemp spinbox in Light window
3. remove unused ui file (light_mainwindonw.ui)

### CODE REVIEW

File: src/common/utils.py [hunk 1]
```diff
@@ -129,6 +115,91 @@ def generate_random_discriminator():
...
+        slider_stylesheet = """
+QSlider::handle:horizontal {
+    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
+        stop:0 #eee, stop:1 #ccc);
+    border: 1px solid #777;
+    width: 14px;
+    margin-top: -3px;
+    margin-bottom: -3px;
+    border-radius: 7px;
+}
```
Comment (junyong-sim) [inline src/common/utils.py:142]: slider_stylesheet = """
            QSlider::handle:horizontal {
                background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                    stop:0 #eee, stop:1 #ccc);
                border: 1px solid #777;
                width: 14px;
                margin-top: -3px;
                margin-bottom: -3px;
                border-radius: 7px;
            }
            QSlider::handle:horizontal:hover {
                background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                    stop:0 #fff, stop:1 #ddd);
                border: 1px solid #444;
            }
            """
please, modify code intent.

File: src/things/light.py [hunk 2]
```diff
@@ -11,7 +11,8 @@
 EVENT_TOGGLE = 1
 EVENT_DIMMING = 2
 EVENT_COLOR = 3
-
+EVENT_SPINBOX_DIMMING = 4
+EVENT_SPINBOX_COLOR = 5
```
Comment (junyong-sim) [inline src/things/light.py:16]:
```
EVENT_DIMMING = 2
EVENT_COLOR = 3
```
Can we able to remove the events above ? I think EVENT_SPINBOX_DIMMING and EVENT_SPINBOX_COLOR are able to replace them.

**Review by junyong-sim — COMMENTED** [review]
Very Good UI !!!
I added some comments. Please, check them.

File: src/things/light.py (hunk 2 above)
Comment (dooho-lee) [reply src/things/light.py:16]: EVENT_SPINBOX_DIMMING is used differently than EVENT_DIMMING at New line 225. EVENT_SPINBOX_COLOR also too.

File: src/common/utils.py (hunk 1 above)
Comment (dooho-lee) [reply src/common/utils.py:142]: when use """ """, Indent is also included in string value. I thought it was unnecessary. What is better code readability and unnecessary tab inclusion?

Comment (dooho-lee) [reply src/common/utils.py]: adjust indent of slider UI stylesheet string in Utils (commit e2c7fe3) for code readability.

* if there are a lot of stylesheet code, we will seperate it with other python file later

**Review by junyong-sim — APPROVED** [review]
LGTM

**Pull request merged**